portfolio/src/main/webapp/script.js

```diff
@@ -61,19 +57,34 @@ function getComments() {
     for (i = 0; i < comments.length; i++) {
       console.log(i);
       dataListElement.appendChild(
-        createListElement(comments[i]));
+        createCommentElement(comments[i]));
     }
   });
 }
 
 /** Creates an <li> element containing text. */
-function createListElement(text) {
-  const liElement = document.createElement('li');
-  liElement.innerText = text;
-  return liElement;
+function createCommentElement(comment) {
```

Comment:
rename from createListElement to createCommentElement since now the function will make a div box with multiple pieces of information. Formerly, it only held lines of text (no nickname or timestamp)